src/main/java/uk/gov/di/ipv/cri/common/library/service/ConfigurationService.java

```diff
@@ -37,33 +41,52 @@ public enum SSMParameterName {
 
     @ExcludeFromGeneratedCoverageReport
     public ConfigurationService() {
-        this.clock = Clock.systemUTC();
-        this.ssmProvider = ParamManager.getSsmProvider();
-        this.secretsProvider = ParamManager.getSecretsProvider();
-        this.parameterPrefix =
-                Objects.requireNonNull(
-                        System.getenv("AWS_STACK_NAME"), "env var AWS_STACK_NAME required");
+        this(
+                ParamManager.getSsmProvider(
+                        SsmClient.builder().httpClient(UrlConnectionHttpClient.create()).build()),
```

Comment:
please test if the following amend works in your stack:

```
ParamManager.getSsmProvider(
                        SsmClient.builder()
                                .credentialsProvider(EnvironmentVariableCredentialsProvider.create())
                                .httpClient(UrlConnectionHttpClient.create())
                                .region(Region.of(System.getenv("AWS_REGION")))
                                .build())
```

Reply:
It works b'cos I originally had this `.region(Region.EU_WEST_2)` of course the above is more flexible. But I removed it. Did want to add it b'cos considered it out of scope